.gitignore

```diff
@@ -5,6 +5,9 @@ src/generator/AutoRest.NodeJS.Tests/AcceptanceTests/*.js
 ## Ignore user-specific files, temporary files, build results, etc.
 AutoRest/**/Templates/*.cs
 src/**/Templates/*.cs
+src/generator/**/Templates/Rest/Common/*.cs
```

Comment:
wouldn't 
` src/**/Templates/**/*.cs ` work to do this globally?  we're gonna have a ton of ignore lines if we don't